test/unit/csv_exporter_test.rb

```diff
@@ -26,4 +26,15 @@ class CsvExporterTest < ActiveSupport::TestCase
       result.next
     end
   end
+
+  test 'accepts custom column headers' do
+    result = CsvExporter.export(Host::Managed, [:id], ['My Lovely Header'])
```

Comment:
In most cases there is no need to rename the headers, so I feel that requiring a title for every column would complicate both the parameters and the implementation.